{"name": "Проверяем, что нельзя изменить данные пользователя без авторизации", "status": "passed", "start": 1731119424569, "stop": 1731119424726, "uuid": "48c04664-3327-4ec3-a7d8-8e61305d53d9", "historyId": "afa8f7ec805bfcd0fe50a9b721b0b25f", "testCaseId": "afa8f7ec805bfcd0fe50a9b721b0b25f", "fullName": "tests.test_change_user_data.TestChangeUserData#test_change_data_unauthorized_returns_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_change_user_data"}, {"name": "subSuite", "value": "TestChangeUserData"}, {"name": "host", "value": "DESKTOP-0KFG2SR"}, {"name": "thread", "value": "12036-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_change_user_data"}]}